main.nf

```diff
@@ -631,14 +624,17 @@ if (params.tenx_tgz || params.bam) {
       is_aligned_channel_id = "${channel_id}__${is_aligned}"
       umis_per_cell = "${is_aligned_channel_id}__n_umi_per_cell.csv"
       good_barcodes = "${is_aligned_channel_id}__barcodes.tsv"
-      """
-      bam2fasta count_umis_percell \\
-          --filename ${reads} \\
-          --min-umi-per-barcode ${tenx_min_umi_per_cell} \\
-          --cell-barcode-pattern '${tenx_cell_barcode_pattern}' \\
-          --molecular-barcode-pattern '${tenx_molecular_barcode_pattern}' \\
-          --write-barcode-meta-csv ${umis_per_cell} \\
-          --barcodes-significant-umis-file ${good_barcodes}
+
+      """  
+        if [[ `gzip -l \$(realpath $reads) | awk 'NR==2 {print \$2}'` ne 0 ]]; then
```

Comment:
@snafees is this right, I am getting an error, is there any other way to evaluate this variable and compare to 0

Reply:
fixed it! :) I was missing a `-` before `ne`

Reply:
sorry i just saw your earlier comment. I see what `ne` is doing there now. Do add your comments next to this line as to why this is needed and it'll be good!